pages/index.tsx

```diff
@@ -39,7 +39,11 @@ const Home: NextPage<IHomeProps> = ({ posts }) => {
         posts={posts}
         categoryFilter
       />
-      {/* <BlogSection title="BROWSE ALL" variant="grid" posts={posts} /> */}
+      <BlogSection
+        title="MOST POPULAR"
+        variant={BlogSectionVariants.mostPopular}
+        posts={posts}
+      />
```

Comment:
Let's just remember that this section, Most Popular, should be visible before the Browse All section.

Right now, they are inverted.